setup.py

```diff
@@ -1,11 +1,32 @@
+import pathlib
+
 from setuptools import find_packages, setup
 
+import versioneer
+
+
+REPO_ROOT = pathlib.Path(__file__).parent
+
+with open(REPO_ROOT / "README.md", encoding="utf-8") as f:
+    README = f.read()
+
+REQUIREMENTS = [
+    "numpy",
+    "gym",
+    "pandas",
+    "pyarrow",
+]
 
 setup(
```

Comment:
Reckon this need setting a `>=` for python version also?

Reply:
yeah i think it has to be in a diff setup arg `python_requires` - we test on 3.8, but I've actually just realised that the env I use is 3.7, and its probably better to be more permissive so i'll put it to >=3.7 lmk if you disagree

Reply:
suppose we should probably add 3.7 to the test matrix as well in this case

Reply:
yep all sounds good - do makes the ticket
I think 3.7+ is good